commit 7d3227b24301d0d1cca13c28a3f8e8c42d6e8c6f
Author: Ake Sandgren <ake.sandgren@hpc2n.umu.se>
Date:   Wed Nov 2 12:37:44 2022 +0100

    Don't use include_directories on MPI include paths.
    None of the variables are set anyway when using EB.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dbfa3c85..df5ee61a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -212,14 +212,6 @@ endif(ALTCPU)
 # -------------------------------------------------------------------------------MPI--
 find_package(MPI REQUIRED)
 
-if ("${MPI_CXX_INCLUDE_DIRS}" STREQUAL "")
-	if (NOT "${MPI_CXX_INCLUDE_PATH}" STREQUAL "")
-		include_directories("${MPI_CXX_INCLUDE_PATH}")
-	endif()
-else()
-	include_directories("${MPI_CXX_INCLUDE_DIRS}")
-endif()
-
 message(STATUS "MPI_INCLUDE_PATH : ${MPI_INCLUDE_PATH}")
 message(STATUS "MPI_LIBRARIES : ${MPI_LIBRARIES}")
 message(STATUS "MPI_CXX_INCLUDE_PATH : ${MPI_CXX_INCLUDE_PATH}")
